homeworks/Fransi Kurti/index.css

```diff
@@ -0,0 +1,39 @@
+:root {
+    /* COLOR PALETTE */
+    --primary-black: #232536;
+    --primary-yellow: #ffd050;
+    --primary-purple: #592ea9;
+    --primary-dark-grey: #4c4c4c;
+    --primary-medium-grey: #6d6e76;
+    --primary-light-grey: #f4f4f4;
+    --primary-lavander: #f4f0f8;
+    --primary-light-yellow: #fbf6ea;
+
+    /* FONT WEIGHT */
+    --regular: 400;
+    --bold: 700;
+    --extra-bold: 800;
+
+    /* TYPOGRAPHY */
+    --display-size: 64px;
+    --h1-size: 48px;
+    --h2-size: 36px;
+    --h3-size: 28px;
+    --h4-size: 24px;
+    --h5-size: 20px;
+    --h6-size: 16px;
+}
+
+/* OVERWRITING DEFAULTS */
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    box-sizing: border-box;
+    font-family: 'Sen', sans-serif;
+}
+
+header {
```

Comment:
**CSS is not complete** , make sure to add the rest and when it is looking like the design then commit.